packages/css_colors/pubspec.yaml

```diff
@@ -6,7 +6,7 @@ version: 1.1.1
 
 environment:
   sdk: '>=2.12.0 <3.0.0'
```

Comment:
Should we remove the higher bound for the `sdk` in these as well?

Reply:
Or set to `<4.0.0`

Reply:
That would be a separate PR, both since it's unrelated to this change and would affect more packages.

It's also still not clear to me if that's correct to do yet.